convert2rhel/actions/__init__.py

```diff
@@ -206,25 +211,122 @@ def run(self):
 
         self._has_run = True
 
-    def set_result(self, status=_NO_USER_VALUE, error_id=_NO_USER_VALUE, message=_NO_USER_VALUE):
+    @property
+    def result(self):
+        """
+        Helper method to save result variable in a different variable, so it does
+        not get overwritten by the __init__ method of the Action class
```

Comment:
```suggestion
        Once the :class:`Action` is run, this holds the `result`.  The result is an :class:`ActionResult`
        which holds information of whether the Action succeeded or failed and why.
```

Reply:
The docstring on the `@property` decorated method ends up being used as the docstring for the property itself so it should be written as if it is describing the attribute rather than the method.

Reply:
Noted